core/domain/subscription_services_test.py

```diff
@@ -159,7 +165,8 @@ def test_posting_to_feedback_thread_results_in_subscription(self):
         # The viewer posts a message to the thread.
         message_text = 'text'
         feedback_services.create_thread(
-            'exp_id', 'state_name', self.viewer_id, 'subject', message_text)
+            'exploration', 'exp_id', 'state_name', self.viewer_id, 'subject',
```

Comment:
Probably 'exploration' should be TARGET_TYPE_EXPLORATION instead?

Reply:
Done